formatTest/typeCheckedTests/expected_output/attributes.4.04.0.re

```diff
@@ -0,0 +1,6 @@
+/* Pexp_letexception with attributes */
+let () = {
+  [@attribute]
+  exception E;
```

Comment:
Is this what the expected refmt output should look like? It's the current behavior, so I set it to pass with this result, but if the attribute is supposed to be on the same line then it should probably be fixed and the test modified to reflect that.

Reply:
This looks good for now! Even if we decide it shouldn't be printed this way, it falls outside of the scope of this PR.

Reply:
Personally, I kind of like it this way. It jives with my expectations for annotations being on the line immediately preceding the thing they are annotating.